### DISCUSSION TITLE
Multiple CRM integrations on one form

### DISCUSSION THREAD

**Discussion by gertjanvancakenbergh** [solspace/craft-freeform#389] [Feature Requests]
We have a project where one form needs to send data to multiple CRMs. Right now in Freeform we can only select one CRM per form to map fields to. 

We've made a workaround by creating a custom CRM integration that collects the fields from both CRM's but it's not a very convenient solution.

**Comment by kjmartens** [comment]
This is possible in Freeform 5 now. 🙂
